### ISSUE TITLE
Back / Foward Browser buttons loading HTMX partial content only

### ISSUE THREAD

**Issue by helixaros** [bigskysoftware/htmx#1201]
Hi there,

I have a url "/settings/users/" where it loads the list of users along with some filtering /searching functionality.
When a user is added on the list / removed I have an HX-Trigger, and the list is rerendered (only the list not the whole page).

If I press the back / forward button (with the previous step being adding / removing a user from the list), then it loads only the partial (doesn't load the full page), and therefore the page breaks (as js / css is not loaded). Example attached. I have seen some discussions happening around this topic, but I am not sure of the resolution that I should take to fix this issue.

Thank you in advance.

<img width="717" alt="Screenshot 2023-01-14 at 10 05 01 AM" src="https://user-images.githubusercontent.com/5782958/212462607-9d6146f0-1567-49cf-b02a-5ff2846d9d8d.png">

**Comment by Frohus** [comment]
I'm experiencing the same issue.
I have an htmx search form with hx-push-url on the home page, when submitted htmx makes a swap in the div on the homepage, then when I click on one of the filtered entries it takes me to its page (no htmx) and when I press back button only the previously filtered entries partial template is loaded not the whole page with that partial as expected

**Comment by David-Guillot** [comment]
Hi there,

Might be related to #724 , and #1206 might help.

**Comment by David-Guillot** [comment]
Hi @helixaros @Frohus ,

Can you confirm this issue is gone with latest 1.8.5? 613f7b61d59e8ac4741e02d9f468ca635773ab2d appears to fix it with a configurable (disabled by default) cache-busting query-param.

**Comment by Frohus** [comment]
@David-Guillot just checked, it's still present

**Comment by 1cg** [comment]
You should be able to fix this with a cache busting option as of 1.8.5, but you will need to configure it.

See the documentation here for dealing with this issue:

https://htmx.org/docs/#caching

**Comment by gnat** [comment]
tl;dr:
```js
<script>htmx.config.getCacheBusterParam = true</script>
```
Can just be in your `<head>` somewhere after you load htmx.

**Comment by zyriab** [comment]
Howdy from the future,

I'm late to the party but figured I'd share that info for people from the futurer: 
This is not a bug, just how HTTP works.
Another solution, if you don't want to change of cache works for the entire app is to check if the header `HX-Request: true` is present in your server's endpoint.
If it is, you serve the partial, otherwise, you serve the entire page.
